engine/src/main/java/com/redhat/cloud/notifications/transformers/BaseTransformer.java

```diff
@@ -11,23 +11,44 @@
 @ApplicationScoped
 public class BaseTransformer {
 
-    public JsonObject transform(Action action) {
-        return toJsonObject(action);
-    }
+    // JSON property names' definition.
+    public static final String ACCOUNT_ID = "account_id";
+    public static final String APPLICATION = "application";
+    public static final String BUNDLE = "bundle";
+    public static final String CONTEXT = "context";
+    public static final String EVENT_TYPE = "event_type";
+    public static final String EVENTS = "events";
+    public static final String METADATA = "metadata";
+    public static final String ORG_ID = "org_id";
+    public static final String PAYLOAD = "payload";
+    public static final String TIMESTAMP = "timestamp";
 
-    protected JsonObject toJsonObject(Action action) {
+    /**
+     * Transforms the given action into a {@link JsonObject}.
+     * @param action the {@link Action} to transform.
+     * @return a {@link JsonObject} containing the given action.
+     */
+    public JsonObject toJsonObject(final Action action) {
         JsonObject message = new JsonObject();
-        message.put("bundle", action.getBundle());
+
+        message.put("account_id", action.getAccountId());
```

Comment:
```suggestion
        message.put(ACCOUNT_ID, action.getAccountId());
```
Same for the other fields :smiley:

Reply:
Oh, you're right... Changed!

Reply:
Done!